Author		: Doug Nazar <nazard@dragoninc.ca>
Date		: Tue, 27 Feb 2007 13:52:15 -0500
Message-ID	: <00d101c75aa0$66420b60$32c62220$@ca>
Status		: pushed to netdev maintainer for inclusion, confirmed working
Description     : fixes frequent unaligned accesses in ip_rcv() and ip_fast_csum()

diff --git a/drivers/net/tulip/tulip_core.c b/drivers/net/tulip/tulip_core.c
index 5a35354..e3774a5 100644
--- a/drivers/net/tulip/tulip_core.c
+++ b/drivers/net/tulip/tulip_core.c
@@ -67,7 +67,7 @@ const char * const medianame[32] = {
 
 /* Set the copy breakpoint for the copy-only-tiny-buffer Rx structure. */
 #if defined(__alpha__) || defined(__arm__) || defined(__hppa__) \
-	|| defined(__sparc_) || defined(__ia64__) \
+	|| defined(__sparc__) || defined(__ia64__) \
 	|| defined(__sh__) || defined(__mips__)
 static int rx_copybreak = 1518;
 #else

